src/tests/control/test_checkins.py

```diff
@@ -260,7 +261,10 @@ def test_checkins_list_ordering(client, checkin_list_env, order_key, expected):
 @pytest.mark.django_db
 @pytest.mark.parametrize("query, expected", [
     ('status=&item=&user=', ['A1Ticket', 'A1Mascot', 'A2Ticket', 'A3Ticket']),
+    ('status=0&item=&user=', ['A1Mascot', 'A2Ticket']),
```

Comment:
this is the same as four lines below, did you mean to add something else?

Reply:
Thanks, I think just copy-paste error, it is now testing all possible status filters